### PR TITLE
feat: Add index stylesheets to each MDC Web package

### PR DESCRIPTION
**Pull request by abhiomkar**
Project import generated by Copybara.

Author: @nex3

### CODE REVIEW

File: testing/dom/index.ts [hunk 1]
```diff
@@ -33,14 +33,3 @@ export const html =
       result += strings[strings.length - 1];
       return result;
     };
-
-/**
- * @returns HTMLElement for given HTML content in string format.
- */
-export function getFixture(content: string): HTMLElement {
```
Comment (abhiomkar) [inline testing/dom/index.ts]: Should revert this before merge.

**Comment by abhiomkar** [comment]
@nex3 mdc-image-list & mdc-layout-grid are missing in this PR. This is expected since these 2 packages are not synced internally.

Can we run the migrator on these 2 packages in a separate PR? We don't need to import these.

Thanks!

**Comment by nex3** [comment]
Done: https://github.com/material-components/material-components-web/pull/5546

**Comment by joyzhong** [comment]
LGTM based on cl/292403730 review

**Pull request merged**